sql/api/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3911,6 +3911,43 @@ object functions {
   def encode(value: Column, charset: String): Column =
     Column.fn("encode", value, lit(charset))
 
+  /**
+   * Returns true if the input is a valid UTF-8 string, otherwise returns false.
+   *
+   * @group string_funcs
+   * @since 4.0.0
+   */
+  def is_valid_utf8(str: Column): Column =
```

Comment:
You should follow naming conventions and use the camelCase style, see https://github.com/databricks/scala-style-guide?tab=readme-ov-file#naming-convention

Reply:
are you sure? going through `functions.scala`, I only see underscore naming, and not camelCase